imports/plugins/core/orders/client/components/orderSummary.js

```diff
@@ -149,15 +156,24 @@ class OrderSummary extends Component {
         <div className="order-summary-form-group">
           <strong data-i18n="orderShipping.shipTo">Ship to</strong>
           <div className="invoice-details">
-            <strong>Phone: </strong>{profileShippingAddress.phone}
+            <strong>Phone: </strong>{profileShippingAddress && profileShippingAddress.phone}
           </div>
         </div>
 
         <div style={{ marginTop: 4 }}>
-          <span>{profileShippingAddress.fullName}</span>
-          <br/><span>{profileShippingAddress.address1}</span>
-          {profileShippingAddress.address2 && <span><br/>{profileShippingAddress.address2}</span>}
-          <br/><span>{profileShippingAddress.city}, {profileShippingAddress.region}, {profileShippingAddress.country} {profileShippingAddress.postal}</span>
+          <span>{profileShippingAddress && profileShippingAddress.fullName}</span>
+          <br/>
+          <span>{profileShippingAddress && profileShippingAddress.address1}</span>
+          {profileShippingAddress && profileShippingAddress.address2 &&
+            <span><br/>{profileShippingAddress.address2}</span>
+          }
+          <br/>
+          <span>
+            {profileShippingAddress && profileShippingAddress.city},&nbsp;
+            {profileShippingAddress && profileShippingAddress.region},&nbsp;
```

Comment:
I'm actually more curious about the need for `&nbsp;` here. Does React not respect spaces here?

Reply:
@spencern I added the `&nbsp;` because the addition of `profileShippingAddress &&`s made the line too long, and I had to break it into different lines, therefore line 172 and 173 end up being two strings without space in between them.

But if I did this:
> You could have setup profileShippingAddress to pass an empty object if undefined and then just used profileShippingAddress.property everywhere.

then I wouldn't have to change the line/add `&nbsp;`s.